kgforge/specializations/models/rdf_model.py

```diff
@@ -107,13 +107,16 @@ def schema_id(self, type: str) -> str:
         except KeyError:
             raise ValueError("type not found")
 
-    def validate(self, data: Union[Resource, List[Resource]], execute_actions_before: bool, type_: str) -> None:
+    def validate(self, data: Union[Resource, List[Resource]], execute_actions_before: bool, type_: str,
+                 debug: bool) -> None:
```

Comment:
Very small detail, but the Model class's signature for this method has debug with default value False, and this one doesn't. It creates a bit of a mismatch (IDE complains but no errors)

Reply:
Changed.